src/locales/fi.json

```diff
@@ -53,6 +56,22 @@
     }
   },
   "form": {
+    "gdpr": {
+      "berthPrivacyStatementLink": {
+        "text": "Venepaikkojen rekisteriseloste (pdf)"
+      },
+      "openCityProfilePrivacyStatementLink": {
+        "text": "Helsinki-profiilin rekisteriseloste",
+        "url": "#"
+      },
+      "cityOfHelsinkiPrivacyStatementLink": {
+        "text": "Helsingin kaupungin tietosuojakäytäntö",
+        "url": "#"
```

Comment:
I'm guessing we don't have these urls yet?

Reply:
Nope and I'm not sure if the pages exists at all, because the Open city profile is not in production yet. 😞